File: Net/SchoolRegistry/Models/Student.cs
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Bson;

namespace SchoolRegistry.Models;

public class Student
{

    [BsonId]
    [BsonRepresentation(BsonType.ObjectId)]
    public string? Id { get; set; }

    public string name { get; set; } = null!;

    public string lastName { get; set; } = null!;

    public string phone { get; set; } = null!;

    public string email { get; set; } = null!;

    public DateTime dateOfBirth { get; set; }

    [BsonRepresentation(BsonType.ObjectId)]
    public string? _classId { get; set; }

    [BsonRepresentation(BsonType.ObjectId)]
    public string? _guardianId { get; set; }

}

File: Node/db/dbmodels/student.js
const mongoose = require('mongoose');

const StudentSchema = new mongoose.Schema({
  name: { type: String, trim: true, minlength: 3 },
  lastName: { type: String, trim: true },
  phone: { type: String },
  email: { type: String },
  dateOfBirth: { type: Date },
  _classId: { type: mongoose.Types.ObjectId, required: true },
  _guardianId: { type: mongoose.Types.ObjectId, required: true },
}, { versionKey: false });

const Student = mongoose.model('Student', StudentSchema);

module.exports = Student;